v63003/u_inref/gtm8795.csh

```diff
@@ -0,0 +1,36 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+source $gtm_tst/com/gtm_test_setbgaccess.csh
+$gtm_tst/com/dbcreate.csh mumps 1>&create.out
+if ($status) then
+	echo "create failed"
+endif
+$MUPIP SET -region DEFAULT -journal=enable,on,before -replication=on
```

Comment:
Why is this command needed? If one runs the subtest with -replic option, then dbcreate automatically takes care of replication. Which means we need this subtest to be run with and without -replic option. In the no -replic option run, we can enable journaling (before or nobefore) by the SETJNL value that way dbcreate automatically creates the journal files.